src/components/ButtonDetails/ButtonDetails.jsx

```diff
@@ -0,0 +1,23 @@
+import { Typography } from "@mui/material";
+import { buttonDetailStyle } from "../../buttonDelivery.styles";
+import { BsLayoutTextSidebar } from "react-icons/bs";
+import Button from "../Button/Button";
+
+const ButtonDetails = () => {
+  return (
+    <Button
+      endIcon={
+        <BsLayoutTextSidebar style={{ color: "#00000075", fontSize: "16px" }} />
+      }
+      sx={{
+        color: "#000",
+        fontWeight: "500",
+        fontSize: 16,
+      }}
+    >
+      Details
+    </Button>
+  );
+};
+
+export default ButtonDetails;
```

Comment:
Wouldn't it be better to make this a generic/shared button and reuse it across the application?
This is the same as de Actions button. The only thing that changes is the text.

Reply:
ok.